### PR TITLE
Lw8 var6

### PR DESCRIPTION
**Pull request by Mackasin**

### CODE REVIEW

**Comment by otkisyan** [comment]
Why do you print each element of stream in some filter methods? I think it would be better here if you will at first collect it, as we were doing in previous lab work

File: src/main/java/org/nuos/laboratory/lw8/logic/Houses.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        houses.stream()
+                .filter(house -> house.getNumberOfRooms() == numberOfRooms && house.getFloor()>=minFloor && house.getFloor()<=maxFloor)
+                .forEach(System.out::println);
+    }
+
+    public void filterBySquare(double square) {
+        houses.stream()
+                .filter(house -> house.getSquare()>square)
+                .sorted(Comparator.comparingInt(House::getFloor))
+                .sorted(Comparator.comparingDouble(House::getSquare).reversed())
```
Comment (otkisyan) [inline src/main/java/org/nuos/laboratory/lw8/logic/Houses.java]: I don't understand why do you use .sorted here twice. The task is: "список квартир, які мають площу, що перевищує задану в порядку спадання площі. Якщо площа однакова – то в порядку зростання поверху"

So I think we can replace it with
```java
public void filterBySquare(double square){
    houses.stream(){
                    .filter(house -> house.getSquare() > square)
                    .sorted(Comparator.comparingDouble(House::GetSquare).reversed()
                                  .thenComparingInt(House::getFloor))
                     .collect(Collectors.toList());
}
```

**Comment by Mackasin** [comment]
maybe u right

**Comment by Mackasin** [comment]
why did u close this pull re?

**Review by otkisyan — APPROVED** [review]

**Pull request merged**